lib/rubocop/cop/rails/active_support_on_load.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Use Active Support lazy load hooks to patch Rails core classes, so they are not forcibly loaded early.
```

Comment:
Can you add `@safety` note?
e.g. https://github.com/rubocop/rubocop-rails/blob/v2.15.2/lib/rubocop/cop/rails/safe_navigation_with_blank.rb#L9-L20